benchmark/source_map/source-map-cache.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const common = require('../common.js');
+const assert = require('assert');
+const path = require('path');
+const fs = require('fs');
+
+const bench = common.createBenchmark(
+  main,
+  {
+    operation: [
+      'findSourceMap-valid',
+      'findSourceMap-generated-source',
+    ],
+    n: [1e5],
+  },
+);
+
+function main({ operation, n }) {
+  const Module = require('node:module');
+
+  Module.setSourceMapsSupport(true, {
+    generatedCode: true,
+  });
+  const validFileName = path.resolve(
+    __dirname,
+    '../../test/fixtures/test-runner/source-maps/line-lengths/index.js',
+  );
+
+  const fileContent = fs.readFileSync(validFileName, 'utf8');
+  const sourceWithGeneratedSourceMap = fileContent + `\n//# sourceURL=${validFileName}`;
```

Comment:
https://github.com/nodejs/node/blob/214e4db60ef55f1a59c93336ab956371b0b058ed/lib/internal/source_map/source_map_cache.js#L386

To test properly, this should be a different file name, avoiding collision.

```suggestion
  const sourceWithGeneratedSourceMap = fileContent + `\n//# sourceURL=${validFileName}`;
```

Reply:
Hey, I had to load another file to avoid that collision while also adding valid magic comments, thanks!